lib/shortcuts/glue-orc-table.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const GlueTable = require('./glue-table');
+
+class GlueOrcTable extends GlueTable {
+  constructor(options = {}) {
+    let { Parameters } = options;
+    const { Location } = options;
+
+    const required = [Location];
+    if (required.some((variable) => !variable))
+      throw new Error('You must provide a Location');
+
+    const TableType = 'EXTERNAL_TABLE';
+    const InputFormat = 'org.apache.hadoop.hive.ql.io.orc.OrcInputFormat';
+    const OutputFormat = 'org.apache.hadoop.hive.ql.io.orc.OrcOutputFormat';
+    const SerdeInfo = {
+      SerializationLibrary: 'org.apache.hadoop.hive.ql.io.orc.OrcSerde',
+      Parameters: { 'serialization.format': '1' }
+    };
+    Parameters = Object.assign(
+      { EXTERNAL: 'TRUE', 'orc.compress': 'SNAPPY' },
```

Comment:
Orc files can be uncompressed or use a different compression strategy. Imo this should be left configurable but default to being uncompressed.

Reply:
I don't think `EXTERNAL: 'TRUE'` is required

Reply:
supported compression formats: https://docs.aws.amazon.com/athena/latest/ug/compression-formats.html

Reply:
I removed default values for any Parameters. They're still accepted if the caller provides them.